readme.md

```diff
@@ -135,23 +135,23 @@ $ npm -v
 
 * **Question:** What is `path` and where does it come from?
 
-* **Your Answer:**
+* **Your Answer: provides utilities for working with file and directory paths, it returns the current working directory
```

Comment:
In this case, `path` is a module that you are given access to via Node. There are _methods_ on `path` that will give you access to the current working directory.